### DISCUSSION TITLE
More code font options

### DISCUSSION THREAD

**Discussion by almson** [community/community#3332] [Mobile]
I'd like more font options in the code preferences section. I already have font size set to minimal, but I'd like it smaller. Also support for my favorite fonts, like Fira Code, would be nice.

Edit: Forgot to mention I'm on iOS, iPhone 12 mini (yes, I have good vision).

**Comment by DrStoop** [comment]
totally agree with @almson, especially smaller font sizes would strongly increase usability and productivity!

**Comment by shailist** [comment]
I mainly need an option to use a monospace font in the code view, my phone's default font looks trash when looking at code

**Comment by adigitoleo** [comment]
@almson Hi, can you direct me to this font size setting? I can't see it (on PC).

**Reply by shailist** [reply]
At least for Android:
`Settings > General > Code Options > Override system font size`

**Reply by adigitoleo** [reply]
Hmm, doesn't seem to be exposed except for on Android, [from what I can tell](https://stackoverflow.com/questions/7372428/changing-the-default-github-code-font). Hopefully we get it for the PC version at some point.

**Comment by CarlGWatts** [comment]
I'd like the option to select a proportional-width font (like the standard font on Android) for Code Viewing.  I switched to programming with proportional-width fonts long ago.  They even make better use of the screen width on mobile apps.  Don't force to me to view code in a monospace font!

**Reply by hossinhabeeb119** [reply]
Czech

**Comment by hossinhabeeb119** [comment]
> I'd like more font options in the code preferences section. I already have font size set to minimal, but I'd like it smaller. Also support for my favorite fonts, like Fira Code, would be nice.
> 
> 
> 
> Edit: Forgot to mention I'm on iOS, iPhone 12 mini (yes, I have good vision).

**Reply by hossinhabeeb119** [reply]
Sfax

**Reply by hossinhabeeb119** [reply]
> > I'd like more font options in the code preferences section. I already have font size set to minimal, but I'd like it smaller. Also support for my favorite fonts, like Fira Code, would be nice.
> > do
> > 
> > fs
> > Edit: Forgot to mention I'm on iOS, iPhone 12 mini (yes, I have good vision).
> 
>
